src/position.h

```diff
@@ -46,9 +51,49 @@ class Position {
 
 };
 
-Piece create_piece(Color side, PieceType ptype);
+inline Piece create_piece(Color side, PieceType ptype) {
+	return (6 * side + ptype);
+}
 
 extern void parse_UCI_move(Position& pos, string& str);
-extern bool square_on_board(Square s);
 
+// Takes a file and rank and returns their 120 based square index
+inline Square FR2SQ(File f, Rank r) {
+	return ((f + 21) + (r * 10));
+}
+
+// Converts a 120 based square to a 64 based square index
+inline Square to64(Square s) {
+	return s < 0 || s >= 120 ? OFFBOARD : sq120_to_64[s];
```

Comment:
Note: this has changed to return `OFFBOARD` when s is not in range (previous functionality undefined)